src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -278,12 +276,6 @@ public void error(ExecutorDriver driver, String message) {
     log.error(this.getClass().getName() + ".error: " + message);
   }
 
```

Comment:
Sure, for a more graceful shutdown, you could do two things: 1) change the executor to a TERMINATING state so that it refuses any other runTasks, etc. and 2) try to gracefully kill the tasks (SIG_INT, wait, SIG_KILL) so that the tasks themselves have time to shut down cleanly (complete file transfers, etc.?)